src/test/java/itm/oss/splitter/AppTest.java

```diff
@@ -0,0 +1,99 @@
+package itm.oss.splitter;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.math.BigDecimal;
+
+import org.junit.jupiter.api.DisplayName;
+import java.io.ByteArrayInputStream;
+
+public class AppTest {
+
+    // JUnit 5 기본 구조
+    @Test
+    void testApp_Placeholder() {
+        assertTrue(true); 
+    }
```

Comment:
The current test method testApp_Placeholder() does not show what it is testing.
It only works as a simple placeholder and does not explain its purpose.

Giving it a clearer name will make the test easier to understand.

```suggestion
    @Test  
    void testAppInitialization() {  
        assertTrue(true);  
    }

Reply:
Thanks for your suggestion.
The name 'testApp_Placeholder()' wasn't clear it's our mistake..!
I've renamed the method to testAppInitialization as you suggested.
Please review and approve if it looks to good.

Reply:
Naming is much clearer than before!
Thank you for your checks.